### ISSUE TITLE
Add tests for guidelines

### ISSUE THREAD

**Issue by jannis-baum** [hpi-dhc/PharMe#249]
- [ ] update unit tests after #213
- [ ] consider integration tests
  - if they are necessary: write them

**Comment by dieknolle3333** [comment]
As far as I can tell, there are no unit tests to add. All methods that could be tested are either private or require interaction with the repo.

**Comment by dieknolle3333** [comment]
For the unit tests we need a seperate database, right? So I assume #333 should be reopened?

**Comment by jannis-baum** [comment]
> For the unit tests we need a seperate database, right? So I assume #333 should be reopened?

@dieknolle3333 If so, feel free to reopen the issue and set up a separate database for the testing environment☺️

**Comment by dieknolle3333** [comment]
So after looking into it again, I personally came to the conclusion that this is unnecessary. The only method that should really be tested is GuidelinesService > supplementSheetData(), and that is already done in the end-to-end tests. All other methods are private, only call very simple methods in the repository or are already tested in the end-to-end tests.
That's why I propose to close the issue

**Comment by jannis-baum** [comment]
Closing as per previous comment
